test/animations.jl

```diff
@@ -110,13 +110,19 @@ end
     end
 end
 
+function disp_text(t, p)
+    setfont("Helvetica", 24)
+    settext(t, p)
+end
+
```

Comment:
Error here due to not all systems possibly possessing this font. Not sure how to get around this.

Reply:
I just changed it to setting the radius of a circle instead.